README.md

```diff
@@ -23,46 +25,60 @@ or
 ```
 conda install -c conda-forge mne-base matplotlib mne-qt-browser
 ```
+
 ## Usage
 
-Import mne-python
+In the following example, we'll read M/EEG data from the MNE `sample` dataset
+and plot it using the `mne-qt-browser`.
+
 ```python
+from pathlib import Path
 import mne
+
+sample_dir = mne.datasets.sample.data_path()
+raw_path = Path(sample_dir) / 'MEG' / 'sample' / 'sample_audvis_raw.fif'
+raw = mne.io.read_raw(raw_path)
+
+mne.viz.set_browser_backend('pyqtgraph')  # Enable mne-qt-browser backend
+raw.plot(block=True)
 ```
-To **use the new backend once**, set `pyqtgraph` as backend with 
-```python
-mne.viz.set_browser_backend("pyqtgraph")
-```
-or to **set it permanently** with
-```python
-mne.set_config('MNE_BROWSE_BACKEND', 'pyqtgraph')
-```
 
-Then load and plot your Raw-data, e.g. by using:
+If the plot is not showing, search for solutions in the
+[troubleshooting](#troubleshooting) section below.
+
+This will use the `mne-qt-browser` for the current Python session. If you
+want to make this change permanent so you don't have to use the
+`set_browser_backend()` anymore, run the following line to modify your MNE
+configuration file:
```

Comment:
```suggestion
This will use the `mne-qt-browser` for the current Python session. If you
want to make this change permanent so you don't have to use the
`set_browser_backend()` each time after restarting Python, run the following
line to modify your MNE configuration file:
```